src/services/OswServiceTaskMemMonitor.cpp

```diff
@@ -51,16 +51,16 @@ void OswServiceTaskMemMonitor::printStats() {
   Serial.println("B");
 
   Serial.print("heap (high):\t");
-  Serial.print(ESP.getHeapSize() - this->heapHigh);
+  Serial.print((ESP.getHeapSize() + ESP.getPsramSize()) - this->heapHigh);
```

Comment:
It seems to be... Not correct - the high value should always be higher than the current (or at least equal).

Take a look here for the high-watermakrs: https://github.com/espressif/arduino-esp32/blob/5458df0a5435db0914d24770865a93e6d35bdab3/cores/esp32/Esp.h#L73

I guess we have to add an `getMinFreePsram()` to https://github.com/Open-Smartwatch/open-smartwatch-os/blob/d0f2a1f4d56091eebabf36c3bef450ad44b1af87/src/services/OswServiceTaskMemMonitor.cpp#L14